HealthcareSystemFrontends/GraphicalEditor/MainWindow.xaml.cs

```diff
@@ -151,17 +152,17 @@ public MainWindow()
             // uncomment only the first time you start the project in order
             // to populate DB with start data
             InitializeDatabaseData initializeDatabaseData = new InitializeDatabaseData();
-            initializeDatabaseData.InitiliazeData();
+            // initializeDatabaseData.InitiliazeData();
+
+            /*  EquipementService equipementService = new EquipementService();
+              List<EquipmentWithRoomDTO> result = equipementService.GetEquipmentWithRoomForSearchTerm("bed");
+              foreach(EquipmentWithRoomDTO res in result)
+              {
+                  Console.WriteLine(res.IdEquipment);
+                  Console.WriteLine(res.RoomNumber);
+                  Console.WriteLine("---");S
+              } */
```

Comment:
Did you mean to delete this or will it be needed for something in the future? It looks like it was just for quick testing purposes, right?

Reply:
it was for testing and wil be deleted when we connect it with front